tests/suites/0_stateless/17_altertable/17_0005_alter_table_modify_column_type.result

```diff
@@ -9,7 +9,13 @@ c	INT	NO	0
 ERROR 1105 (HY000) at line 1: Internal. Code: 1001, Text = invalid float literal while evaluating function `to_float32('a')`.
 ERROR 1105 (HY000) at line 1: UnknownColumn. Code: 1058, Text = Cannot find column b.
 0	1
-ERROR 1105 (HY000) at line 1: Internal. Code: 1001, Text = invalid float literal while evaluating function `to_float32('a')`.
+ERROR 1105 (HY000) at line 1: Internal. Code: 1001, Text = error: 
+  --> SQL:1:1
+  |
+1 | alter table test_modify_column_type.c modify column a float default 'a'
+  | ^^^^^^^^^^^^^^^ invalid float literal while evaluating function `to_float32('a')`
```

Comment:
the error hint is not related to the sql statement.

Reply:
```
CREATE table b(a int default 'a');

1 | CREATE table b(a int default 'a')
  | ^^^^^^^^^^^^^ invalid digit found in string while evaluating function `to_int32('a')`
```

the same behaivor when creating table.